code/src/index.css

```diff
@@ -5,9 +5,158 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: #FFD831;
+  box-sizing: border-box;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.image-wrapper {
+  display: flex;
+  justify-content: center;
+  margin: 1.5rem auto 1.5rem auto;
+  max-width: 520px;
+  min-width: 350px;
+}
+
+.logo {
+  width: 100%;
+}
+
+main {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+  min-width: 340px;
+  max-width: 500px;
+  font-size: 1rem;
+  margin: 1.5rem auto 1rem auto;
+  padding: 10px 12px 5px 10px;
+  background-color: #FFD831;
+}
+
+.form-section {
+  border: black solid 3px;
+  border-radius: 3px;
+  background-color: #A6A6A6;
+  box-shadow: 0.5rem 0.5rem rgb(31, 31, 31);
+  padding: 1rem;
+  margin: 1rem;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+}
+
+.label {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
+  "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
+  sans-serif;
+  display: flex;
+  flex-direction: column;
+  font-weight: 500;
+}
+
+.thought-input {
+  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
```

Comment:
Do you need all these fonts? Perhaps make a --var in the beginning so you won't have to write as much code for each.